videointelligence/video_quickstart/main.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START video_quickstart]
+
+// Sample video_quickstart uses the Google Cloud Video Intelligence API to label a video.
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/golang/protobuf/ptypes"
+
+	// Imports the Google Cloud Video Intelligence API client package.
+	video "cloud.google.com/go/videointelligence/apiv1"
+	"golang.org/x/net/context"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Creates a client.
+	client, err := video.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	op, err := client.AnnotateVideo(ctx, &videopb.AnnotateVideoRequest{
+		InputUri: "gs://demomaker/cat.mp4",
+		Features: []videopb.Feature{
+			videopb.Feature_LABEL_DETECTION,
+		},
+	})
+	if err != nil {
+		log.Fatalf("Failed to start annotation job: %v", err)
+	}
+
+	resp, err := op.Wait(ctx)
+	if err != nil {
+		log.Fatalf("Failed to annotate: %v", err)
+	}
+
+	// Only one video was processed, so get the first result.
+	result := resp.GetAnnotationResults()[0]
```

Comment:
How about:
```
annotations := resp.GetAnnoationResults()
if len(annotations) == 0 {
   log.Fatalf("No annotations found")
}
// Only one video was processed, so get the first
results := annotations[0]
```

Reply:
I think this is guaranteed to be len==1.

Reply:
Oh okay, gotcha, thanks for that update. In that case, you've got my vote Mr. President!

Reply:
I'll wait for @remi (Cloud ML API expert) to confirm.